internal/empleado.go

```diff
@@ -0,0 +1,5 @@
+package health_scheduler
+
+type Empleado string
+
+var TurnosAsignados map[*Empleado][]Turno
```

Comment:
... una de las cuales es una variable global *que no debe usarse nunca*. Aunque tenga ámbito de paquete.

Reply:
Como puede ver en [https://github.com/abbonno/healthScheduler/pull/10#discussion_r1871243016](el comentario), @abbonno me sugirió que fuese una variable, pero es cierto que, es mucho más necesario que sea un `type`

Reply:
Y tú seguiste un comentario que no tiene nada que ver con los issues ni con las historias de usuario, creando trabajo extra para todos, incluyéndome a mi. Repetidamente os he dicho que los comentarios no son para sugerencias, sino que sirven para plantear alternativas o entender mejor qué es lo que se ha planteado. Igual que una HU no es un mandao, un milestone no es un mandao, un comentario a un PR *tampoco es un mandao*, y esto tenéis que entenderlo los dos.

Reply:
Finalmente se opta por modelar TurnosAsignados como una entidad [bde5ecf]